test_matrix.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from Matrix import Matrix
+
+
+class Test_matrix(unittest.TestCase):
```

Comment:
how about adding tests that check your exceptions sanity?
for example, craeting a `Matrix` with `str` and not `tuple` -> should throw an exception.

Reply:
How can i check if some action (like creating Matrix with string) actually throws one?